src/pages-and-resources/discussions/app-config-form/AppConfigFormApplyButton.jsx

```diff
@@ -4,14 +4,14 @@ import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { StatefulButton } from '@edx/paragon';
 
 import messages from './messages';
-import { SAVING } from './data/slice';
+import { SAVING } from '../data/slice';
 import { AppConfigFormContext } from './AppConfigFormProvider';
 
 function AppConfigFormApplyButton({ intl }) {
-  const status = useSelector(state => state.discussions.appConfigForm.status);
+  const saveStatus = useSelector(state => state.discussions.saveStatus);
```

Comment:
Saving gets its own status variable now, since we combined our app list and app config form reducers again.  "status" is whether or not the GET data has been loaded from the server.